File: SDKs/plaid.com/2020-09-14_1.20.6/typescript/src/sdk/models/operations/banktransfereventlist.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class BankTransferEventListRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "request, media_type=application/json" })
  request: shared.BankTransferEventListRequest;
}


export class BankTransferEventListResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  bankTransferEventListResponse?: Record<string, any>;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  error?: Record<string, any>;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: SDKs/plaid.com/2020-09-14_1.20.6/python/src/sdk/models/operations/banktransfereventlist.py
import dataclasses
from typing import Any,Optional
from ..shared import banktransfereventlistrequest as shared_banktransfereventlistrequest


@dataclasses.dataclass
class BankTransferEventListRequest:
    request: shared_banktransfereventlistrequest.BankTransferEventListRequest = dataclasses.field(metadata={'request': { 'media_type': 'application/json' }})
    

@dataclasses.dataclass
class BankTransferEventListResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    bank_transfer_event_list_response: Optional[dict[str, Any]] = dataclasses.field(default=None)
    error: Optional[dict[str, Any]] = dataclasses.field(default=None)
    
